source/System.cpp

```diff
@@ -303,6 +308,11 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 				object.message = &UNINHABITEDPLANET;
 		}
 	}
+	
+    // Make sure that the stellarWindStrength and luminosity are not game brakeingly low or high.
+    stellarWindStrength = std::max(0.25, stellarWindStrength);
+    luminosity = std::max(0.5, luminosity);
+    luminosity = std::min(2.0, luminosity);
```

Comment:
If you allow negatives then there is no point in bounding it.  A minimum wind makes sense as there is some gas anywhere in galactic space, and black holes are a _good_ place to collect gas because their gravity condenses it.